modules/tinymce/src/core/test/ts/browser/SelectionOverridesTest.ts

```diff
@@ -15,9 +17,32 @@ describe('browser.tinymce.core.SelectionOverridesTest', () => {
     disable_nodechange: true,
     entities: 'raw',
     indent: false,
+    content_style: 'body { margin: 16px; }',
     base_url: '/project/tinymce/js/tinymce'
   }, [ Theme ]);
 
+  const getScrollTop = (editor: Editor) => Scroll.get(TinyDom.document(editor)).top;
+
+  const assertSelectionIsCaretBlock = (editor: Editor, caretValue: 'before' | 'after') => {
+    const selectedNode = editor.selection.getNode();
+    assert.isTrue(isCaretContainerBlock(selectedNode));
+    assert.equal(selectedNode.getAttribute('data-mce-caret'), caretValue);
+  };
+
+  const selectBesideContentEditable = (editor: Editor, contentEditableElm: HTMLElement, clickBefore: boolean) => {
```

Comment:
Maybe instead of a boolean this could be `'left' | 'right'`?

Reply:
Have updated the function to use`'before' | 'after'` and do the cart block check as well